docs_src/microservices/application/services/AppServiceConfigurable.md

```diff
@@ -228,23 +620,23 @@ Required:
 
     There are many optional functions and parameters provided in this profile. See the [complete profile](https://github.com/edgexfoundry/app-service-configurable/blob/{{edgexversion}}/res/mqtt-export/configuration.yaml) for more details
 
-### sample
+#### sample
 
 Sample profile with all available functions declared and a sample pipeline. Provided as a sample that can be copied and modified to create new custom profiles. See the [complete profile](https://github.com/edgexfoundry/app-service-configurable/blob/{{edgexversion}}/res/sample/configuration.yaml) for more details
 
-### functional-tests
+#### functional-tests
 
 Profile used for the TAF functional testing  
 
-### external-mqtt-trigger
+#### external-mqtt-trigger
 
 Profile used for the TAF functional testing  of external MQTT Trigger
 
-## What if my input data isn't an EdgeX Event ?
+### What if my input data isn't an EdgeX Event ?
 
 The default `TargetType` for data flowing into the functions pipeline is an EdgeX Event DTO. There are cases when this incoming data might not be an EdgeX Event DTO. There are two setting that configure the TargetType to non-Event data.
 
-### Raw TargetType
+#### Raw TargetType
 
  In these cases the `Pipeline` can be configured using `TargetType="raw"` to set the `TargetType` to be a byte array/slice, i.e. `[]byte`. The first function in the pipeline must then be one that can handle the `[]byte` data. The **compression**,  **encryption** and **export** functions are examples of pipeline functions that will take input data that is `[]byte`. 
 
```

Comment:
there is typo in line 652:  Alogrithm --> Algorithm

Reply:
Good ctahc!